src/content/advanced/configure-dex-in-your-cluster/index.md

```diff
@@ -246,6 +265,8 @@ __Note__: When applying the example in the snippet above, please replace the `<C
 
 Then submit the resource to the management API and the App operator will manage it to make the actual installation and configuration. You can log in now into the cluster API with your identity provider using the login endpoint that Dex creates for you. By default, it will be `https://login.<CLUSTER>.<BASEDOMAIN>`.
 
+__Warning__: In this explanation we have assumed you have running an ingress controller and cert manager in you cluster in order to make dex available for the callback request made by your identity provider in a secure way. If you supply custom certificates when deploying dex then you can skip cert manager installation. Both apps are offered in our managed app catalog.
```

Comment:
```suggestion
__Warning__: It is assumed that you have an ingress controller and cert-manager running in your cluster in order to make dex available for the callback request made by your identity provider securely. If you supply custom certificates when deploying dex, then you can skip cert-manager installation. Both of these apps are offered in our managed app catalog.
```

Reply:
Can we also link to docs about installing the ingress controller and cert-manager (if they exist)?

Reply:
done